client/packages/invoices/src/StockOut/utils.tsx

```diff
@@ -406,8 +406,9 @@ export const UnitQuantityCell = (props: CellProps<DraftStockOutLine>) => (
       (props.rowData.stockLine?.packSize ?? 1)
     }
     id={getPackQuantityCellId(props.rowData.stockLine?.batch)}
-    decimalLimit={2}
     min={0}
+    // We're not allowing non-integer values (decimalLimit=0), as you can only
+    // issue whole units
```

Comment:
Fix for #6090. Remove `decimalLimit` prop which forces an integer.

Reply:
idk if we want to fix this one because of the use case I mentioned in issue 🤔 maybe revert this change and just close the bug? 🤔